### PR TITLE
Run on latest point releases of Ruby

### PR DESCRIPTION
**Pull request by maclover7**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ env:
 
 rvm:
   - 2.0.0
-  - 2.1.4
-  - 2.2.0
+  - 2.1.7
+  - 2.2.3
```
Comment (tute) [inline .travis.yml]: Should we remove the patch versions? Suggestion:

```
   - 2.0
   - 2.1
   - 2.2
``

Comment (mcmire) [inline .travis.yml]: Oh, I didn't know you could do this. If this works, then yeah, I think this makes sense.

**Comment by maclover7** [comment]
:+1:

**Comment by tute** [comment]
:+1:

**Comment by maclover7** [comment]
ping @mcmire

**Comment by tute** [comment]
If it's ok with you @mcmire, I can rebase and merge branches in this project when we both agree (and you didn't get to before).

**Comment by mcmire** [comment]
@tute Okay! That would be helpful. Thank you.

By the way this looks good. :+1:

**Comment by tute** [comment]
Thank you @maclover7! :clap:

**Comment by maclover7** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request closed without merging**